### PR TITLE
Jcaccamo/surface placement rel to scene

### PR DESCRIPTION
**Pull request by jared-esri**
@ldanzinger Ready for review. Thanks!

Updated surface placement sample with new surface placement mode, relative to scene.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.cpp [hunk 1]
```diff
@@ -59,11 +59,16 @@ void Surface_Placement::componentComplete()
...
+
+  // Create scene layer from the Brest, France scene server.
+  ArcGISSceneLayer* sceneLayer = new ArcGISSceneLayer(QUrl("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer"), this);
+  scene->operationalLayers()->append(sceneLayer);
   //! [Create Scene API snippet]
 
   // set an initial viewpoint
-  Camera camera(53.05, -4.01, 1115, 299, 88, 0);
-  Viewpoint viewpoint(53.05, -4.01, 1115, camera);
+  Point initialViewPoint(-4.45968, 48.3889, 37.9922);
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.cpp:69]: all of these can be const

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.cpp [hunk 2]
```diff
@@ -102,38 +112,47 @@ void Surface_Placement::addGraphicsOverlays()
 
 void Surface_Placement::addGraphics()
 {
-  // create point for the graphics with z value of 1000
-  Point point(-4.04, 53.06, 1000, SpatialReference::wgs84());
+  // create point for the scene related graphic with a z value of 0
+  Point sceneRelatedPoint(-4.4610562, 48.3902727, 0, SpatialReference::wgs84());
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.cpp:116]: const

File: ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/Surface_Placement.qml [hunk 3]
```diff
@@ -38,22 +38,26 @@ Rectangle {
                 }
             }
 
+            ArcGISSceneLayer {
+                url: "https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/"
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/Surface_Placement.qml:42]: may want to remove trailing forward slash. We generally don't format like this

**Comment by ldanzinger** [comment]
@jared-esri since this is being reworked, should we close out the PR for now?

**Comment by jared-esri** [comment]
closing PR as the sample is being reworked.

**Pull request closed without merging**